cmd/device-worker/main.go

```diff
@@ -145,8 +146,24 @@ func main() {
 
 	reg := registration2.NewRegistration(&hw, deviceOs, dispatcherClient, configManager, hbs, wl, dataMonitor, metricsStore, systemMetricsWatcher)
 
+	// defined how to connect to hosts
+	ansiblePlaybookConnectionOptions := &options.AnsibleConnectionOptions{ //FIXME
+		Connection: "local",
+		User:       "test_user",
+	}
+	// defined which should be the ansible-playbook execution behavior and where to find execution configuration.
+	ansiblePlaybookOptions := &playbook.AnsiblePlaybookOptions{ //FIXME
```

Comment:
What exactly need to be fixed her?

Reply:
I'm not sure which Fields should be set and what are the correct values.